python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_faiss.py

```diff
@@ -36,9 +36,17 @@
 pytestmark = pytest.mark.skipif(_DISABLED, reason="vpdq not installed")
 
 if not _DISABLED:
-    EXAMPLE_META_DATA = {"example_video"}
-    VIDEO1_META_DATA = {"video1"}
-    VIDEO2_META_DATA = {"video2"}
+    EXAMPLE_META_DATA = dict(
+        {
+            "hash_type": "vpdq",
+            "video_id": 5,
+        }
+    )
```

Comment:
Nit: this is exactly the same as 
```suggestion
    EXAMPLE_META_DATA = {
        "hash_type": "vpdq",
        "video_id": 5,
    }
```